e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js

```diff
@@ -91,7 +91,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
         });
     });
 
-    it('Post Reactions option for Members', () => {
+    it('MM-T1544 Post Reactions option for Members', () => {
```

Comment:
Same here. It has 2 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

Reply:
Consolidated steps in TM4J